lib/tasks/redirect_legacy_email_signups.rake

```diff
@@ -0,0 +1,31 @@
+desc "Redirect legacy email signups"
+task redirect_legacy_email_signups: :environment do
+  specialist_sectors_info = Topic.where(state: %w[draft published]).where.not(parent_id: nil).map do |ss|
```

Comment:
Also, it'd be useful if the commit or a comment said why we need a parent_id of nil. I didn't know myself so had to go work it out from the frontend.

Reply:
I've added a comment which should make that part of the query more clear.

As for:
> Do we need to worry about redirects for draft topics?

I thought it'd be good to add these in for drafts too so draft and published specialist sectors email-signup paths are at least consistent in the router. Otherwise we'd have dangling /email-signup routes for the draft sectors, what do you think?

Reply:
> > Do we need to worry about redirects for draft topics?
> 
> I thought it'd be good to add these in for drafts too so draft and published specialist sectors email-signup paths are at least consistent in the router. Otherwise we'd have dangling non redirect /email-signup routes for the draft sectors, what do you think?

Sure that makes sense but I'd not worry too much about router consistency since that is really the domain of communication between content store and router-api (and it'd only be in the draft stack). I think we end up in a slightly weirder situation if we present the drafts to the Publishing API as it adds in a pile of content items that will never be published to live and may linger around to cause a surprise one day.

Reply:
Okay dokay that's a fair point. 

I've removed drafts from the query so the only redirects getting created are for published specialist sectors.